### DISCUSSION TITLE
Learning about how copilotkit works.

### DISCUSSION THREAD

**Discussion by SuibhneOFoighil** [CopilotKit/CopilotKit#837] [General]
Hey everyone, I was just curious how the framework connected fronted state to Langraph state. Is there a write-up anywhere so I can understand?

**Comment by Muhammed2013h-blip** [comment]
@[WhatsApp.Chat.-.+90.531.944.53.38.zip](https://github.com/user-attachments/files/29152188/WhatsApp.Chat.-.%2B90.531.944.53.38.zip)

**Reply by Muhammed2013h-blip** [reply]
![image](https://github.com/user-attachments/assets/71becdfe-393c-4dc1-a592-02044c2318f2)
